File: trunk/poll/jteam/src/ilsrep/poll/model/Choice.java
package ilsrep.poll.model;

import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlRootElement;

/**
 * The "choice" element.
 * 
 * @author Taras Kostiak
 * 
 */
@XmlRootElement(name = "choice")
public class Choice {

    /**
     * Choice identifier.
     */
    protected String id = null;

    /**
     * Choice text.
     */
    protected String name = null;

    /**
     * @see #id
     */
    @XmlAttribute(required = true)
    public String getId() {
        return id;
    }

    
    /**
     * @see #id
     */
    public void setId(String id) {
        this.id = id;
    }

    
    /**
     * @see #name
     */
    @XmlAttribute(required = true)
    public String getName() {
        return name;
    }

    
    /**
     * @see #name
     */
    public void setName(String name) {
        this.name = name;
    }

}

File: trunk/poll/sharpteam/SharpTeam.PollClient/Model/Choice.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml;
using System.Xml.Serialization;

namespace Ilsrep.PollApplication.Model
{
    [Serializable]
    public class Choice
    {
        [XmlAttribute("id")] public int id;
        [XmlAttribute("name")] public string choice;
        [XmlIgnore] public Poll parent;
    }
}
